hw4_numpy/numpy_challenge.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import numpy as np
+
+if __name__ == "__main__":
+        ar1 = np.array([1, 2, 21])
+        ar2 = np.arange(1,19).reshape(3,-1)
+        ar3 = np.zeros(5)
+
+def matrix_multiplication(h,i):
+    #s = np.matmul(h, i)
+    s = np.dot(h,i)
+    return s
```

Comment:
1. Странный нейминг
2. Лучше не оставлять задокументированный код
```suggestion
def matrix_multiplication(matrix1,matrix2):
    return np.dot(matrix1,matrix2)
```